jenkins/v2/cortx-ha.spec

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
create a directory in jenkins with name rpm then v1, v2 inside in it.

Reply:
done